src/hardware/mixer.cpp

```diff
@@ -931,16 +843,21 @@ void MIXER_Init(Section* sec) {
 	mixer.mastervol[0]=1.0f;
 	mixer.mastervol[1]=1.0f;
 
+	/* Calculate blocksize from requested latency to nearest power of 2 */
+	const auto clamped_latency = clamp(static_cast<int>(mixer.latency), 1, 100);
+	auto blocksize = static_cast<uint16_t>(
+	        next_power_of_two(mixer.freq * clamped_latency / 1000));
```

Comment:
Is freq a double?
The math feels more natural if we do:

Sample rate/s divided by 1000 ms/s to get samplerate per ms. 

Then latency in ms multiplied by rate/ms to get bytes. 

(Ie: 14 * 48000 / 1000). 

 Just thinking out loud..

Reply:
it's `uint32_t`, hence the multiply first followed by divide. No need for floating point.

Reply:
Right on; yup.. left to right math means it works out the same in both 👍 (resolving!)